st2common/in-requirements.txt

```diff
@@ -19,6 +19,8 @@ requests
 retrying
 semver
 six
+# Need our fork with various bug fixes in etcd drivers
+git+https://github.com/StackStorm/tooz.git@etcd3_driver_fixes#egg=tooz
```

Comment:
Per https://github.com/StackStorm/stackstorm-ha/pull/61#pullrequestreview-224058430, looks like we'll also need `etcd3gw` to make `etcd3+http` entrypoint (https://docs.openstack.org/tooz/latest/user/drivers.html#etcd3-gateway) work out of the box